frontend/src/types.ts

```diff
@@ -4370,6 +4370,7 @@ export type HogFunctionInvocationGlobals = {
     >
 }
 
+// TODO: move to schema.ts
```

Comment:
TODO?

Reply:
I was planning to do it in a follow up PR, but yeah, I can fix it here, done